pyproject.toml

```diff
@@ -43,9 +43,9 @@ django-auth-ldap = {version = "~4.1.0", optional = true}
 # Caching with Redis
 django-cacheops = "~6.0"
 # Base functionality for task scheduling with Celery
-django-celery-beat = "~2.2.1"
+django-celery-beat = "~2.3.0"
```

Comment:
Change/potential breakage: https://github.com/celery/django-celery-beat/pull/525

Reply:
Also https://github.com/celery/django-celery-beat/pull/528 - now requires upgrading to `django-timezone-field >= 5.0`. Will investigate if this is feasible.

Reply:
Spent far too much time digging into a migration error resulting from this update, but finally opened https://github.com/mfogel/django-timezone-field/issues/96